internal/driver/driver.go

```diff
@@ -50,15 +50,15 @@ func (s *Driver) Initialize(lc logger.LoggingClient, asyncCh chan<- *dsModels.As
 		panic(fmt.Errorf("unsupport GPIO ABI interface: %v", config.Abi_driver))
 	}
 	s.config = config
-	s.lc.Info(fmt.Sprintf("Interface: %v", config.Abi_driver))
-	s.lc.Info(fmt.Sprintf("ChipSelected: %v", config.Chip_selected))
+	s.lc.Infof(fmt.Sprintf("Interface: %v", config.Abi_driver))
```

Comment:
when you use `Infof`, you don't need to invoke `fmt.Sprintf`
for example,
`s.lc.Infof("Interface: %v", config.Abi_driver)`

Reply:
Thanks for your comment, we will fix this place in the upcoming commit